crates/subspace-networking/src/peer_info.rs

```diff
@@ -52,7 +52,7 @@ pub enum PeerInfo {
     /// DSN farmer.
     Farmer {
         /// Peer info data.
-        cuckoo_filter: CuckooFilterDTO,
+        cuckoo_filter: Arc<CuckooFilterDTO>,
```

Comment:
I read through changes of this PR and have not found why does it need to be wrapped in an `Arc`, am I missing something?